Jenkinsfile

```diff
@@ -57,8 +59,6 @@ ansiColor('xterm') {
             }
 
             sh 'git checkout upstream/master'
-            sh 'git reset --hard && git clean -f'
```

Comment:
Should we keep this in to make sure we're guaranteed a clean folder?

Reply:
I'm hoping the `git gc` above does most of that for us, but let's keep an eye on it. If it does need a real clean and reset, we'll want to add it to the sdk as well.